docs/user/summary.md

```diff
@@ -0,0 +1,1115 @@
+# NetBox Import Summary
+
+This document describes the summary of mapping from NetBox 3.6 to Nautobot 2.1.0. Mappings for other versions may vary based on available models and fields.
+
+The summary is generated when running the following command:
+
+```shell
+nautobot-server import_netbox \
+    --dry-run \
+    --summary \
+    --field-mapping \
+    --bypass-data-validation \
+    /tmp/netbox_data.json
+```
+
+The summary is divided into several sections as described below.
+
+## DiffSync Summary
+
+The first section is the import summary. It shows a summary from the underlying `DiffSync` library.
+
+```
+= Import Summary: ==============================================================
+- DiffSync Summary: ------------------------------------------------------------
+create: 5865
+update: 1
+delete: 0
+no-change: 4
+skip: 119
+```
+
+## Nautobot Models Summary
+
+This section shows the number of objects that are imported for each Nautobot model.
+
+```
+- Nautobot Models Summary: -----------------------------------------------------
+extras.customfield: 1
+extras.status: 6
+extras.role: 18
+dcim.locationtype: 6
+dcim.location: 95
+tenancy.tenantgroup: 1
+tenancy.tenant: 11
+users.user: 6
+circuits.circuit: 29
+circuits.circuittermination: 45
+circuits.circuittype: 4
+circuits.provider: 9
+circuits.providernetwork: 1
+dcim.devicetype: 15
+dcim.consoleport: 41
+dcim.consoleporttemplate: 11
+dcim.device: 72
+dcim.devicebay: 14
+dcim.frontport: 912
+dcim.frontporttemplate: 120
+dcim.interface: 1586
+dcim.interfacetemplate: 300
+dcim.powerfeed: 48
+dcim.poweroutlet: 104
+dcim.poweroutlettemplate: 8
+dcim.powerpanel: 4
+dcim.powerport: 75
+dcim.powerporttemplate: 26
+dcim.rack: 42
+dcim.rearport: 630
+dcim.rearporttemplate: 73
+dcim.cable: 108
+ipam.prefix: 94
+contenttypes.contenttype: 119
+extras.taggeditem: 72
+auth.group: 1
+dcim.rackreservation: 2
+dcim.manufacturer: 15
+ipam.ipaddress: 180
+ipam.vlan: 63
+virtualization.cluster: 32
+virtualization.virtualmachine: 180
+virtualization.vminterface: 720
+dcim.devicebaytemplate: 14
+dcim.platform: 3
+dcim.virtualchassis: 4
+ipam.vrf: 6
+ipam.routetarget: 12
+ipam.rir: 8
+ipam.vlangroup: 7
+extras.tag: 26
+virtualization.clustertype: 6
+virtualization.clustergroup: 4
+```
+
+## Validation Issues
+
+This section shows validation issues that were found during the import process grouped by the Nautobot content type with a total summary at the end.
+
+These issues are not necessarily errors, but may be warnings or other issues that should be reviewed. See the [Data Validation and Error Handling chapter](./app_use_cases.md#data-validation-and-error-handling) for details.
+
+```
+- Validation issues: -----------------------------------------------------------
+. dcim.powerfeed: 48 ...........................................................
```

Comment:
you could trim a bit the verbose output to make it more readable

Reply:
Trimmed.